biosynseq/visualize.py

```diff
@@ -0,0 +1,14 @@
+"""Visualize synthetic sequences using t-SNE and UMAP."""
+import os
+import numpy as np
+import matplotlib.pyplot as plt # need to import matplotlib before pandas
+import pandas as pd
+from pathlib import Path
+from mdlearn.utils import plot_scatter
+from typing import List, Dict
+
+from Bio import SeqIO, SeqUtils, SeqRecord
+from Bio.SeqUtils.IsoelectricPoint import IsoelectricPoint
+
+os.environ["CUDA_VISIBLE_DEVICES"] = "2"
```

Comment:
Setting `CUDA_VISIBLE_DEVICES` in the script can be problematic, instead let's leave it up to the user running the code. You can safely remove this line.

Reply:
Still need to remove this one